lib/pgtk/stash.rb

```diff
@@ -34,11 +34,20 @@ class Pgtk::Stash
   #
   # @param [Object] pgsql PostgreSQL connection object
   # @param [Hash] stash Optional existing stash to use (default: new empty stash)
-  # @param [Loog] loog Logger for debugging (default: null logger)
-  def initialize(pgsql, stash = { queries: {}, tables: {} })
+  # @param [Hash] options Optional options for internal tuning
+  # @option [Integer] refresh Interval in seconds for recalculate stale queries
+  # @option [Integer] top Number of queries to recalculate
+  # @option [Concurrent::ReentrantReadWriteLock] entrance Lock for write internal state
+  # @option [Concurrent::FixedThreadPool] threadpool ThreadPool for execution tasks in background
+  # @option [Loog] loog Logger for debugging (default: null logger)
+  def initialize(pgsql, stash = { queries: {}, tables: {} }, **options)
     @pgsql = pgsql
     @stash = stash
-    @entrance = Concurrent::ReentrantReadWriteLock.new
+    @refresh = options[:refresh] || 5
+    @top = options[:top] || 100
```

Comment:
@Yegorov wouldn't it be more obvious to declare them explicitly in the signature of the constructor:

```
def initialize(pgsql, stash = { queries: {}, tables: {} }, refresh: 5, top: 100, loog: Loog::NULL)
```

Reply:
@yegor256 Yes, in this case (if you do not pass `entrance` and `threadpool`) you can do it this way too.
Otherwise I get this error from Rubocop:
```
lib/pgtk/stash.rb:44:17: C: Metrics/ParameterLists: Avoid parameter lists longer than 6 parameters. [7/6]
  def initialize(pgsql, stash = { queries: {}, tables: {} }, entrance: nil, threadpool: nil, refresh: 5, top: 100, loog: Loog::NULL)
```
Need to resolve the comment above